v3/docs/03-runtime/e-storage/index.mdx

```diff
@@ -151,10 +152,18 @@ pub(super) type SomeNMap<T> = StorageNMap<
 Notice that the map's storage items specify [the hashing algorithm](#hashing-algorithms) that will
 be used.
 
+### QueryKindTrait
+
+The [`QueryKindTrait`](/rustdocs/latest/frame_support/storage/types/trait.QueryKindTrait.html) generic of storage determines 
+how the storage should be handled when there is no value in storage. With [`OptionQuery`](/rustdocs/latest/frame_support/storage/types/struct.OptionQuery.html), when no value is in storage 
+the method get will return `None`. With [`ValueQuery`](/rustdocs/latest/frame_support/storage/types/struct.ValueQuery.html), 
+when no value is in storage the `get` method will return the value configured with the `OnEmpty` generic. 
+When you have a specific default value to configure, use `ValueQuery`.
+
 ### Visibility
 
 In the examples above, all the storage items except `SomePrivateValue` are made public by way of the
-`pub(super)` keyword. Blockchain storage is always publicly
+`pub` keyword. Blockchain storage is always publicly
```

Comment:
but in the example above it is still written `pub(super)` maybe you want to write `pub` in the example above too no ?